### ISSUE TITLE
tables

### ISSUE THREAD

**Issue by hecatia-elegua** [InputUsername/zola-hook#6]
I just noticed these are missing.
Not really sure how to do those myself, any pointers? Or do you want to add them?

**Comment by InputUsername** [comment]
> I just noticed these are missing. Not really sure how to do those myself, any pointers? Or do you want to add them?

Yes, table styles are not implemented but I'm open to adding it to the theme. If you're familiar with SASS (https://sass-lang.com/), I'd be happy to help you create a pull request to implement it.

**Comment by InputUsername** [comment]
@hecatia-elegua I took a stab at implementing this myself in #7, it looks like this:

![Light theme](https://github.com/InputUsername/zola-hook/assets/3883165/fafad96d-5a64-4a29-95cc-67880b401612)
  
![Dark theme](https://github.com/InputUsername/zola-hook/assets/3883165/04e6cc3b-5ee0-4b6d-98d3-d01b882085f1)

Would this work for you?

**Comment by hecatia-elegua** [comment]
[I now just fumbled around with some CSS](https://hecatia-elegua.github.io/blog/no-more-bit-fiddling/#how-bit-fiddling-works-and-why-it-sucks).
But yours is good for tables in general!
